### PR TITLE
A quick copyedit

### PR DESCRIPTION
**Pull request by cyrichardson**
Started out just reading the guide and found a few typos, which led to a few more edits.

### CODE REVIEW

File: _component_design/loading-indicators.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ A variety of spinner icon sizes are available.
 
 {% endcolumn %}
 
+</div>
+
+<div class="hxRow" markdown="1">
```
Comment (nimbinatus) [inline _component_design/loading-indicators.md]: Any reason for adding these here? Typically, there's a left column and a right column within  one row div. This breaks them up so they don't line up properly.

File: _component_design/progress-indicators.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Progress Indicators
 parent: Processing
 layout: component
 category: Components
-usage: The Helix 12-column grid provides structure for website content.
+usage: The Helix 12 Column Grid provides structure for website content.
```
Comment (nimbinatus) [inline _component_design/progress-indicators.md]: I don't think this should be capitalized. The grid is a generic thing, not something specific to Helix. And "12-column" is a descriptor as that's a type of grid...

File: _component_design/progress-indicators.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Progress Indicators
...
 preview-image: components-grid.svg
 need: progress-indicators
 status: unknown
```
Comment (nimbinatus) [inline _component_design/progress-indicators.md]: Also, if the resource is listed as false, the page is not visible to the world. You can skip these as they're still in a state of flux or may be removed completely.

File: _docs/getting-started/design.md [hunk 4]
```diff
@@ -77,4 +77,3 @@ The best way to bring Helix components into your designs is with the Helix Sketc
 
 </div>
 </section>
```
Comment (nimbinatus) [inline _docs/getting-started/design.md]: Any reason for removing the trailing new line? Generally Jekyll likes them.

File: _docs/layout/architecture.md [hunk 5]
```diff
@@ -30,8 +30,8 @@ Application.
 The global layer wraps around the entire Helix-UI and at least some portion of
 it is always visible, regardless of what the user is currently viewing. The
 global layer is best thought about as a collection of offscreen elements that
-will remain hidden until some action is taken by the user. Once an action is
-taken the global layer services up contextually sensitive info and workflows
+remain hidden until some action is taken by the user. After an action is
+taken, the global layer serves contextually sensitive info and workflows
 for managing platform(global) services. View the
```
Comment (nimbinatus) [inline _docs/layout/architecture.md]: I'd add a space before the `(` here. Mind grabbing that for us as you're in here?

File: _docs/layout/demo-options.md [hunk 6]
```diff
@@ -9,12 +9,12 @@ resource: false
```
Comment (nimbinatus) [inline _docs/layout/demo-options.md]: This is a non-public page. No changes needed as this probably will get removed.

File: _docs/layout/global-layer.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: Global Layer
 layout: docs
 category: Layout
 usage: |
-  The global layer is the application layer that allows a user to access
+  The global layer is the application layer that enables a user to access
   global features of the UI.
 preview-image:
 resource: false
```
Comment (nimbinatus) [inline _docs/layout/global-layer.md]: Also a non-public page that may get removed (`resource: false`)

File: _docs/layout/grid.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 title: Grid
 layout: docs
 category: Layout
-usage: The Helix 12-column grid provides structure for website content.
+usage: The Helix 12 Column Grid provides structure for website content.
```
Comment (nimbinatus) [inline _docs/layout/grid.md]: I don't think this should be capitalized. The grid is a generic thing, not something specific to Helix. And "12-column" is a descriptor as that's a type of grid..

File: _docs/layout/grid.md [hunk 9]
```diff
@@ -122,4 +122,3 @@ Resize your browser to see how columns scale and stack on smaller devices.
     </div>
   </div>
   </section>
```
Comment (nimbinatus) [inline _docs/layout/grid.md]: Same question about the removed trailing new line

File: _docs/layout/page-layouts.md [hunk 10]
```diff
@@ -162,4 +162,5 @@ Build the content area using a 12 Column Grid. See [grid](/layout/grid) for more
 -->
 {% endcolumn %}
 </div>
-</section>
\ No newline at end of file
+
```
Comment (nimbinatus) [inline _docs/layout/page-layouts.md]: Why was a new line added here?

File: _docs/layout/page-layouts.md [hunk 11]
```diff
@@ -23,7 +23,7 @@ resource: true
 - Use the main navigation to navigate between the main sections of the
   application. Make sure your application consistently
   uses either a top navigation or a side navigation, but never both. You can
-  use dropdown for subcategories, but be sure to only use one sub-level.
+  use dropdowns for subcategories, but be sure to only use one sub-level.
```
Comment (nimbinatus) [inline _docs/layout/page-layouts.md]: Actually, we should also hyphenate drop-down. We're trying to catch them when we find them.

File: _docs/principles/accessibility.md [hunk 12]
```diff
@@ -131,6 +131,7 @@ Helix minimum contrast standards require the following elements:
 
 The use of space in visual design provides for designs that are both clear and pleasing to the eyes. Space creates relationships, defines hierarchies, and emphasizes content. Always keep these in mind when considering how to reduce visual noise and create visual flow in your design.
 
+
```
Comment (nimbinatus) [inline _docs/principles/accessibility.md]: new line? Please remove.

File: _docs/style/colors.md [hunk 13]
```diff
@@ -89,4 +89,6 @@ sure to use enough contrast between adjacent colors.
   {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#212121" color="#9e9e9e" csscode="Gray 600 on Gray 975" hex="#9E9E9E on #212121" ratio="6.0 : 1" %}
   {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#000000" color="#757575" csscode="Gray 700 on Gray 1000" hex="#757575 on #000000" ratio="4.6 : 1" %}
 </div>
-</section>
\ No newline at end of file
+
```
Comment (nimbinatus) [inline _docs/style/colors.md]: ? new line?

File: _docs/style/iconography.md [hunk 14]
```diff
@@ -15,13 +15,17 @@ helix-ui-javascript: true
 
 {% include toc.html %}
 <section class="static-section"  markdown="1">
-### New Requests
-If you need an additional icon, please submit your request to the
+
+### New requests
+If you need an additional icon, submit your request to the
+
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: ? new line?

File: _docs/style/iconography.md [hunk 15]
```diff
@@ -41,7 +45,9 @@ links or match the color of body text dependent on the background color.
 </section>
 
 <section class="static-section"  markdown="1">
-### Object Icons
+
+### Object icons
+
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: ? new line?

File: _docs/style/iconography.md [hunk 16]
```diff
@@ -80,4 +88,6 @@ gray, or white, corresponding to the color of the body text of the background.
     {% endif %}
   {% endfor %}
 </div>
-</section>
\ No newline at end of file
+
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: ? new line?

File: _docs/style/iconography.md [hunk 17]
```diff
@@ -61,11 +67,13 @@ a list or symbolize types of content.
 </div>
 </section>
 
+
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: ? new line?

File: _docs/style/typography.md [hunk 18]
```diff
@@ -37,7 +37,9 @@ helix-ui-javascript: false
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Page title
-Page titles are the highest level of hierarchy. There should only ever be one on any page.
+
+Page titles are the highest level of hierarchy. There should only be one on any page.
+
```
Comment (nimbinatus) [inline _docs/style/typography.md]: ? new line?

File: _docs/style/typography.md [hunk 19]
```diff
@@ -52,7 +54,9 @@ Page titles are the highest level of hierarchy. There should only ever be one on
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Section title
-Use section titles for titles of page sections. For example, if your page title is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading would use section titles.
+
+Use section titles for titles of page sections. For example, if your page title is **Billing**, and the page has **Overview**, **Trends**, and **Payment methods** section headings, each section heading uses section titles.
+
```
Comment (nimbinatus) [inline _docs/style/typography.md]: ? new line?

**Comment by cyrichardson** [comment]
Thanks, @nimbinatus. Will make changes!

File: _docs/layout/demo-options.md (hunk 6 above)
Comment (cyrichardson) [reply _docs/layout/demo-options.md]: Changes reversed.

**Comment by cyrichardson** [comment]
@nimbinatus I've applied your edits.

File: _docs/style/typography.md [hunk 20]
```diff
@@ -115,7 +117,8 @@ Body text is the default style and should be used across the interface for most
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Subdued style
-You can use the subdued style with any text 14px and taller, and is used to let more important text stand out. The subdued style is not the same as an inactive link or button.
+ts
```
Comment (nimbinatus) [inline _docs/style/typography.md]: Looks like a minor whoops where `ts` got added by mistake.

File: _docs/layout/demo-options.md (hunk 6 above)
Comment (nimbinatus) [reply _docs/layout/demo-options.md]: One missed, but I'm not going to worry about it.

**Comment by cyrichardson** [comment]
Thanks a ton, @nimbinatus! Made that change!

**Comment by cyrichardson** [comment]
Thanks, @DonNateR! I think I got all of Laura's comments addressed.

**Comment by cyrichardson** [comment]
@nimbinatus Please let me know if I didn't get them all.

**Pull request merged**